src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java

```diff
@@ -121,19 +121,6 @@ public AnalysisAjaxController(AnalysisSubmissionService analysisSubmissionServic
 		this.emailController = emailController;
 	}
 
-	/**
-	 * Get all {@link User} generated {@link AnalysisOutputFile} info for principal User
-	 *
-	 * @param principal Principal {@link User}
-	 * @return {@link User} generated {@link AnalysisOutputFile} info
-	 */
-	@RequestMapping(value = "/user/analysis-outputs")
-	@ResponseBody
-	public List<ProjectSampleAnalysisOutputInfo> getAllUserAnalysisOutputInfo(Principal principal) {
-		final User user = userService.getUserByUsername(principal.getName());
```

Comment:
Don't get the user here, do this is the service.  This constructor is out of control (and I know that is not your fault).  You can use the `SecurityContextHolder` to get the current user from within the service.

Reply:
This looks like some code I forgot to remove as its been moved to the AnalysesOutputsAjaxController

Reply:
Method removed in c6f2f6cfa2f5b74494bf05a58d24fd7fb08c35fb